pysite/base_route.py

```diff
@@ -27,6 +31,15 @@ def setup(cls: "RouteView", blueprint: Blueprint):
 
 
 class APIView(RouteView):
+    def validate_key(self, api_key: str):
+        """ Placeholder! """
+        return api_key == os.environ("API_KEY")
+
+    def generate_api_key(n: int):
```

Comment:
`n` should probably be defaulted to something, or maybe not present at all depending on how things go

Reply:
Sure, we can default it to 32 if that sounds reasonable.

Reply:
I'm just gonna hardcode it to 32 for now. not convinced we'll ever need it to be anything else. and also it's missing a `self`, oops.